components/legacy_sidebar/sidebar_channel/sidebar_channel.jsx

```diff
@@ -12,7 +12,7 @@ import {intlShape} from 'utils/react_intl';
 import {trackEvent} from 'actions/telemetry_actions.jsx';
 import * as GlobalActions from 'actions/global_actions';
 import SidebarChannelButtonOrLink from '../sidebar_channel_button_or_link/sidebar_channel_button_or_link.jsx';
-import SidebarTutorialTip from '../sidebar_tutorial_tip.jsx';
```

Comment:
@neallred from a first pass, everything looks great. I think we should revert the changes made in the legacy sidebar code

Reply:
I can do that.